src/styles.css

```diff
@@ -41,3 +41,10 @@ body {
 .cdk-drop-list-dragging .cdk-drag {
   transition: var(--transition);
 }
+
+.centered {
+  position: fixed;
```

Comment:
This appears to be overridden by a `position: relative` on the `mat-spinner` itself. I don't think this is important, and it can be removed. I'll follow up and do this after merging.